migration-guides/0.15-Migration-Guide.md

```diff
@@ -672,6 +672,13 @@ The async `exists` version from `purescript-node-fs` has been removed since the
 **To fix**:
 - use `exists` from the `Sync` module
 
+### Remove `exists` (`purescript-node-fs-aff`)
+
+`exists` has been removed since the underlying `Async.exists` from `purescript-node-fs` has been removed.
+
+**To fix**:
+- use `exists` from the `Sync` module, if needed in an `Aff` context use `liftEffect (Sync.exists path)`.
```

Comment:
`node-fs-aff` doesn't have a `Sync` module. So, perhaps this could be clarified to say:
```markdown
- import the `node-fs` package's `Node.Fs.Sync (exists)` function and wrap in `liftEffect` (e.g. `liftEffect $ Sync.exists path`)
```

Reply:
Done